ov_msckf/src/core/VioManager.cpp

```diff
@@ -567,9 +567,12 @@ void VioManager::do_feature_propagate_update(double timestamp) {
     total_filter_time += (time_total - time_track);
     total_frame_time += time_total;
 
+#ifndef NDEBUG
```

Comment:
I know these changes don't belong in this PR, but we're short on time and this is change that we've been meaning to make for some time.